charts/gardener/gardenlet/templates/clusterrolebinding-managed-istio.yaml

```diff
@@ -1,7 +1,3 @@
-# ManagedIstio feature gate related ClusterRoleBinding.
```

Comment:
The file name and the cluster role binding name still contain `managed-istio`. Do we want to get rid of them, e.g. renaming them to `istio` instead of `managed-istio`?